### PR TITLE
Create role-lead unresponsiveness process

### PR DESCRIPTION
**Pull request by gracenng**
An action item from [1.28 retro](https://bit.ly/k8s128-retro), outline what happens when role lead and shadows are unresponsive

### CODE REVIEW

File: release-team/release-team-removal.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Resignation
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:1]:
```suggestion
# Release Team Removal Process

The release team is reassembled with a new team for each cycle. All team members are automatically removed if they do not apply and are selected for the next release cycle. This process works well, but from time to time a team member may leave the release team in the middle of the release cycle or the release team leaders may decide to part ways early. This document describes the process for this eventuality.

## Resignation
```

Comment (leonardpahlke) [reply release-team/release-team-removal.md:1]: lets add some intro to the document.

File: release-team/release-team-removal.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emiritus Advisor to find a replacement if they would like to step down.
+
+# Removal Process
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:4]:
```suggestion
```

File: release-team/release-team-removal.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emiritus Advisor to find a replacement if they would like to step down.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:2]:
```suggestion
Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor to find a replacement if they would like to step down.
```

File: release-team/release-team-removal.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+# Removal Process
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affect the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:5]:
```suggestion
Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances.
```

File: release-team/release-team-removal.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emiritus Advisor to find a replacement if they would like to step down.
+
+# Removal Process
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affect the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances. 
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by EA (we host 2-3 sessions in different timezone)
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:10]:
```suggestion
- Failure to attend shadow orientation hosted by EA (we host 2–3 sessions in different timezones)
```

File: release-team/release-team-removal.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+- Failure to attend shadow orientation hosted by role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
+- Failure to attend 3 consecutive release meeting without communication to role lead
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:12]:
```suggestion
- Failure to attend 3 consecutive release meetings without communication to role lead
```

File: release-team/release-team-removal.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+- Failure to perform shadow tasks in due time, after 2 discussions with role lead about performance
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on the team's responsibility
+- Violation of CNCF Code of Conduct
+
+Note that communications ahead of time to inform the leads or find alternative arrangements are greatly preferred in these circumstances.
+### Process
+It is important to be kind and patient while trying to resolve the situation.
+- Upon a pattern of inactivity, role lead will notify EA and have initial discussion with shadows about performance.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:20]:
```suggestion
- Upon a pattern of inactivity, the role lead will notify EA and have initial discussion with shadows about performance.
```

File: release-team/release-team-removal.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+- If the shadow is still not contributing to the team, EA and lead will remove shadow from team, and find replacement if appropriate. EA will be responsible to find backfill.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:21]:
```suggestion
- If the shadow is still not contributing to the team, EA and lead will remove the shadow from the team, and find a replacement if appropriate. EA will be responsible for finding backfill.
```

File: release-team/release-team-removal.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+Reasons for removal could be including but not limited to:
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
+- Failure to perform tasks expected of a role lead
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on team
+- Violation of CNCF Code of Conduct
+
+### Process
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
+- Release lead and EA will have initial discussion with role lead
+- If the situation is not improved, release lead and EA will have another discussion with role lead and begin to look for replacement.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:40]:
```suggestion
- If the situation is not improved, the release lead and EA will have another discussion with the role lead and begin to look for a replacement.
```

File: release-team/release-team-removal.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+- If deemed necessary by release lead, EA and sig-release chairs, role lead will be replaced with an experience shadow or someone who previously lead the same role. EA will be responsible to find backfill
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:41]:
```suggestion
- If deemed necessary by the release lead, EA and sig-release chairs, the role lead will be replaced with an experience shadow or someone who previously led the same role. EA will be responsible for finding backfill
```

File: release-team/release-team-removal.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+It is important to be kind and patient while trying to resolve the situation.
+- Upon a pattern of inactivity, role lead will notify EA and have initial discussion with shadows about performance.
+- If the shadow is still not contributing to the team, EA and lead will remove shadow from team, and find replacement if appropriate. EA will be responsible to find backfill.
+
+### Consequences
+- If the current release is the reason the shadow has Kubernetes membership, membership should be revoked
+- Removal of name in release-team document
+- Shadow will not be able to return to the following release. However, they are welcome to apply to releases after that.
+
+# Role Lead
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:28]:
```suggestion
## Role Lead Removal Guideline
```

File: release-team/release-team-removal.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
+- Release lead and EA will have initial discussion with role lead
+- If the situation is not improved, release lead and EA will have another discussion with role lead and begin to look for replacement.
+- If deemed necessary by release lead, EA and sig-release chairs, role lead will be replaced with an experience shadow or someone who previously lead the same role. EA will be responsible to find backfill
+  
+### Consequences
+- Removal of name in release-team document
+- Role lead will need to shadow again and demonstrate sufficient responsibility, and leading ability, before holding another leadership position on the release team
+
+# Appeal
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:47]:
```suggestion
## Appeal
```

File: release-team/release-team-removal.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+- Violation of CNCF Code of Conduct
+
+### Process
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
+- Release lead and EA will have initial discussion with role lead
+- If the situation is not improved, release lead and EA will have another discussion with role lead and begin to look for replacement.
+- If deemed necessary by release lead, EA and sig-release chairs, role lead will be replaced with an experience shadow or someone who previously lead the same role. EA will be responsible to find backfill
+  
+### Consequences
+- Removal of name in release-team document
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:44]:
```suggestion
- Removal of name in the release-team document which documents the release team members of the release cycle
```

File: release-team/release-team-removal.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+- Shadow will not be able to return to the following release. However, they are welcome to apply to releases after that.
+
+# Role Lead
+A role lead has been selected by your previous team lead to supervise, perform the duties of your team 
+and mentor your shadows to be future leads. Failure to perform lead duty can have a lasting impact on the release team.
+Reasons for removal could be including but not limited to:
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
+- Failure to perform tasks expected of a role lead
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on team
+- Violation of CNCF Code of Conduct
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:35]:
```suggestion
- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
```

File: release-team/release-team-removal.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by EA (we host 2-3 sessions in different timezone)
+- Failure to attend shadow orientation hosted by role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
+- Failure to attend 3 consecutive release meeting without communication to role lead
+- Failure to perform shadow tasks in due time, after 2 discussions with role lead about performance
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on the team's responsibility
+- Violation of CNCF Code of Conduct
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:15]:
```suggestion
- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
```

**Review by leonardpahlke — COMMENTED** [review]
I have added some comments, mainly to correct small spelling mistakes.

File: release-team/release-team-removal.md (hunk 1 above)
Comment (npolshakova) [reply release-team/release-team-removal.md:1]: I agree- I think an intro would be helpful. Maybe rephrase slights: 

```
All team members are automatically removed if they do not re-apply and are selected for the next release cycle. While this process generally functions smoothly, there are instances when a team member might leave mid-cycle or the release team leaders may decide to part ways early. 
```

File: release-team/release-team-removal.md (hunk 4 above)
Comment (npolshakova) [inline release-team/release-team-removal.md:5]:
```suggestion
Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affect the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances. 
```

File: release-team/release-team-removal.md (hunk 5 above)
Comment (npolshakova) [inline release-team/release-team-removal.md:10]: Can we write out Emeritus Advisor here or mention the abbreviation in an earlier section?

File: release-team/release-team-removal.md (hunk 6 above)
Comment (npolshakova) [inline release-team/release-team-removal.md:12]: So to clarify, missing consecutive release meetings is fine as long as it's been cleared by the role lead, correct? The different time zones make it a little tricky for some of the shadows to go to all of the release team meetings so it would be good to add this to the shadow application process. It looks like it's missing from the[ release shadows doc](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md) and we could potentially add a point to the [logistics section](https://github.com/kubernetes/sig-release/blob/master/release-team/release-team-selection.md#logistics) in the release team selection as well.

File: release-team/release-team-removal.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+Note that communications ahead of time to inform the leads or find alternative arrangements are greatly preferred in these circumstances.
+### Process
+It is important to be kind and patient while trying to resolve the situation.
```
Comment (npolshakova) [inline release-team/release-team-removal.md:19]: I would add a bullet point to clarify this point `Failure to perform shadow tasks in due time, after 2 discussions with role lead about performance`. I don't think this would immediately trigger reaching out to the EA. I would first reach out to the shadow directly over slack to check in. Personally, if after 2 discussions of the shadow failing to perform tasks on time, then I would only raise it to the EA. 

```suggestion
### Process
It is important to be kind and patient while trying to resolve the situation.
- Notify the shadow in private message over slack of missed deadline and first warning.
- On second occurrence, schedule 1:1 meeting with shadow over slack and discuss performance.
```

File: release-team/release-team-removal.md (hunk 4 above)
Comment (krol3) [reply release-team/release-team-removal.md:5]: At the end of the cycle, the leads of each team could confirm a list of the shadows abandon their duties and be removal from the github release team (https://github.com/kubernetes/sig-release/blob/master/releases/release-1.xx/release-team.md)

File: release-team/release-team-removal.md [hunk 17]
```diff
@@ -0,0 +1,52 @@
+# Release Team Removal Process
+	
+The release team is reassembled with a new team for each cycle. All team members are automatically removed if they do not re-apply and are selected for the next release cycle. While this process generally functions smoothly, there are instances when a team member might leave mid-cycle or the release team leaders may decide to part ways early. 
+
+## Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
+
+
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances.
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:9]:
```suggestion
Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances, but our first priority is a stable, high-quality release.
```

Adding a bit at the end here just because it feels a touch abrupt without.

File: release-team/release-team-removal.md [hunk 18]
```diff
@@ -0,0 +1,52 @@
...
+A role lead has been selected by your previous team lead to supervise, perform the duties of your team 
+and mentor your shadows to be future leads. Failure to perform lead duty can have a lasting impact on the release team.
+Reasons for removal could be including but not limited to:
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
+- Failure to perform tasks expected of a role lead
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on team
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+### Process
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:42]:
```suggestion
- SIGs outside of SIG Release, shadows, leads, or contributors can express concern to release lead or EA.
```

File: release-team/release-team-removal.md [hunk 19]
```diff
@@ -0,0 +1,52 @@
...
+- Release lead and EA will have initial discussion with role lead
+- If the situation is not improved, the release lead and EA will have another discussion with role lead and begin to look for a replacement.
+- If deemed necessary by the release lead, EA and sig-release chairs, the role lead will be replaced with an experience shadow or someone who previously led the same role. EA will be responsible for finding backfill
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:45]:
```suggestion
- If deemed necessary by the release lead, EA and SIG Release chairs, the role lead will be replaced with an experience shadow or someone who previously led the same role. EA will be responsible for finding backfill
```

File: release-team/release-team-removal.md [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+  
+### Consequences
+- Removal of name in release-team document
+- Role lead will need to shadow again and demonstrate sufficient responsibility, and leading ability, before holding another leadership position on the release team
+
+## Appeal
+If the release team member disagrees with their removal decision. They can escalate to SIG-Release chairs via private messaging on Slack.
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:52]:
```suggestion
If the release team member disagrees with their removal decision, they can escalate to SIG Release chairs via private messaging on Slack or by emailing sig-release-leads@kubernetes.io.
```

Maybe emailing all of the SIG Release leads at once is better here? Versus potentially DMing only one of them.

**Review by katcosgrove — COMMENTED** [review]
Some small stuff

File: release-team/release-team-removal.md [hunk 21]
```diff
@@ -0,0 +1,52 @@
+# Release Team Removal Process
+	
+The release team is reassembled with a new team for each cycle. All team members are automatically removed if they do not re-apply and are selected for the next release cycle. While this process generally functions smoothly, there are instances when a team member might leave mid-cycle or the release team leaders may decide to part ways early. 
+
+## Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:6]:
```suggestion
Voluntary resignation is outlined in the [shadow handbook](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md#leaving-the-shadow-program-before-the-release-cycle-ended). The Role Lead should discuss with the Release Lead and the Emeritus Advisor (EA) to find a replacement if they would like to step down.
```

Comment (katcosgrove) [inline release-team/release-team-removal.md:6]:
```suggestion
Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). The role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
```

File: release-team/release-team-removal.md (hunk 17 above)
Comment (leonardpahlke) [reply release-team/release-team-removal.md:9]: 👍

File: release-team/release-team-removal.md [hunk 22]
```diff
@@ -0,0 +1,52 @@
...
+- Upon a pattern of inactivity, role lead will notify EA and have initial discussion with shadows about performance
+- If the shadow is still not contributing to the team, EA and lead will remove the shadow from the team, and find a replacement if appropriate. EA will be responsible for finding backfill.
+
+### Consequences
+- If the current release is the reason the shadow has Kubernetes membership, membership should be revoked
+- Removal of name in release-team document
+- Shadow will not be able to return to the following release. However, they are welcome to apply to releases after that
+
+## Role Lead Removal Guideline
+A role lead has been selected by your previous team lead to supervise, perform the duties of your team
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:33]:
```suggestion
A role lead has been selected by the previous team lead to supervise, perform the duties of the team 
```

File: release-team/release-team-removal.md [hunk 23]
```diff
@@ -0,0 +1,52 @@
...
+- If the shadow is still not contributing to the team, EA and lead will remove the shadow from the team, and find a replacement if appropriate. EA will be responsible for finding backfill.
+
+### Consequences
+- If the current release is the reason the shadow has Kubernetes membership, membership should be revoked
+- Removal of name in release-team document
+- Shadow will not be able to return to the following release. However, they are welcome to apply to releases after that
+
+## Role Lead Removal Guideline
+A role lead has been selected by your previous team lead to supervise, perform the duties of your team 
+and mentor your shadows to be future leads. Failure to perform lead duty can have a lasting impact on the release team.
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:34]:
```suggestion
and mentor the shadows to be future leads. Failure to perform lead duties can have a lasting impact on the release team.
```

File: release-team/release-team-removal.md [hunk 24]
```diff
@@ -0,0 +1,52 @@
...
+Reasons for removal could be including but not limited to:
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:35]:
```suggestion
Reasons for removal could include but are not limited to:
```

File: release-team/release-team-removal.md [hunk 25]
```diff
@@ -0,0 +1,52 @@
...
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
```
Comment (katcosgrove) [inline release-team/release-team-removal.md:36]:
```suggestion
- Failure to attend or send appropriate representation to 2 consecutive release meetings or relevant SIG meetings
```

File: release-team/release-team-removal.md [hunk 26]
```diff
@@ -0,0 +1,52 @@
...
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
+
+
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances.
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by the EA (we host 2-3 sessions in different timezones)
+- Failure to attend shadow orientation hosted by the role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:15]:
```suggestion
- Failure to attend the orientation hosted by the Role Lead (if the Role Lead doesn't offer a timezone-appropriate meeting, the shadow should communicate to the Role Lead about obtaining the provided information async and watching the recording)
```

We have the shadow orientation by the EA. I would not name team sync’s shadow orientation meetings as it might confuse folks.

File: release-team/release-team-removal.md [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+## Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
+
+
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances.
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by the EA (we host 2-3 sessions in different timezones)
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:14]:
```suggestion
- Failure to attend the shadow orientation hosted by the EA (we host 2-3 sessions at the beginning of the release cycle in different timezones)
```

**Review by katcosgrove — COMMENTED** [review]
Most of these are nits

File: release-team/release-team-removal.md [hunk 28]
```diff
@@ -0,0 +1,52 @@
...
+- Failure to attend shadow orientation hosted by the role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
+- Failure to attend 3 consecutive release meetings without communication to the role lead
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:16]:
```suggestion
- Failure to attend 3 consecutive release team meetings without communication to the Role Lead
```

File: release-team/release-team-removal.md [hunk 29]
```diff
@@ -0,0 +1,52 @@
...
+- Failure to perform shadow tasks in due time, after 2-3 discussions with the role lead and the EA about performance
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:17]:
```suggestion
- Failure to perform shadow tasks in due time, after 2-3 discussions with the Role Lead and the EA about performance
```

File: release-team/release-team-removal.md [hunk 30]
```diff
@@ -0,0 +1,52 @@
...
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on the team's responsibility
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+Note that communications ahead of time to inform the leads or find alternative arrangements are greatly preferred in these circumstances.
+### Process
+It is important to be kind and patient while trying to resolve the situation.
+- Upon a pattern of inactivity, role lead will notify EA and have initial discussion with shadows about performance
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:24]:
```suggestion
- Upon a pattern of inactivity, the Role Lead will notify the EA and have an initial discussion with the shadow about their performance.
```

File: release-team/release-team-removal.md [hunk 31]
```diff
@@ -0,0 +1,52 @@
...
+- If the shadow is still not contributing to the team, EA and lead will remove the shadow from the team, and find a replacement if appropriate. EA will be responsible for finding backfill.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:25]:
```suggestion
- If the shadow is still not contributing to the team, the EA and the Release Lead will remove the shadow from the team, and find a replacement if appropriate. The EA will be responsible for finding backfill.
```

File: release-team/release-team-removal.md (hunk 23 above)
Comment (leonardpahlke) [inline release-team/release-team-removal.md:34]:
```suggestion
A Role Lead has been selected to supervise a sub-team within the release team. This role encompasses not only overseeing team activities and contributing to the release cycle but also mentoring shadows throughout the cycle. The work of the Role Leads is pivotal in ensuring a successful release cycle, while simultaneously mentoring shadows to develop them into new contributors within the Kubernetes community. Failure to perform the lead duty can have a lasting impact on the release team.
```

rephrased, ptal.

File: release-team/release-team-removal.md [hunk 32]
```diff
@@ -0,0 +1,52 @@
...
+and mentor your shadows to be future leads. Failure to perform lead duty can have a lasting impact on the release team.
+Reasons for removal could be including but not limited to:
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
+- Failure to perform tasks expected of a role lead
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on team
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+### Process
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
+- Release lead and EA will have initial discussion with role lead
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:43]:
```suggestion
- The Release Lead and the EA will have an initial discussion with the Role Lead
```

File: release-team/release-team-removal.md [hunk 33]
```diff
@@ -0,0 +1,52 @@
...
+- If the situation is not improved, the release lead and EA will have another discussion with role lead and begin to look for a replacement.
```
Comment (leonardpahlke) [inline release-team/release-team-removal.md:44]:
```suggestion
- If the situation does not improve within a week after the first discussion, the Release Lead and the EA will have another discussion with the Role Lead and begin to look for a replacement.
```

within a week?

**Review by leonardpahlke — COMMENTED** [review]
Added a couple of comments about formatting, adding missing a’s and similar, mostly minor stuff—it looks like it's then ready to ship it.  🚀

File: release-team/release-team-removal.md (hunk 33 above)
Comment (gracenng) [reply release-team/release-team-removal.md:44]: it might take more than a week to get signal. this is also more general guideline

File: release-team/release-team-removal.md (hunk 20 above)
Comment (gracenng) [reply release-team/release-team-removal.md]: removed the DM option

**Review by katcosgrove — COMMENTED** [review]
/lgtm

Thanks for taking this on!
